File: support_tools/swigwin-3.0.12/Examples/test-suite/python/smart_pointer_templatevariables_runme.py
from smart_pointer_templatevariables import *

d = DiffImContainerPtr_D(create(1234, 5678))

if (d.id != 1234):
    raise RuntimeError
# if (d.xyz != 5678):
#  raise RuntimeError

d.id = 4321
#d.xyz = 8765

if (d.id != 4321):
    raise RuntimeError
# if (d.xyz != 8765):
#  raise RuntimeError

File: support_tools/swigwin-3.0.12/Examples/test-suite/go/smart_pointer_templatevariables_runme.go
package main

import . "./smart_pointer_templatevariables"

func main() {
	d := NewDiffImContainerPtr_D(Create(1234, 5678))

	if d.GetId() != 1234 {
		panic(0)
	}
	//if (d.xyz != 5678):
	//  panic(0)

	d.SetId(4321)
	//d.xyz = 8765

	if d.GetId() != 4321 {
		panic(0)
	}
	//if (d.xyz != 8765):
	//  panic(0)
}
